sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/aio/__init__.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._digitaltwins_client_async import DigitalTwinsClient
+from .._generated.models._models import ModelData
+from .._generated.models._models import QueryResult
+from .._generated.models._models import PagedModelDataCollection
```

Comment:
The generated models should not be added to the aio namespace.

Reply:
Unless there are async versions

Reply:
Fixed